packages/documentation/src/shared/package-tag.mdx

```diff
@@ -0,0 +1,3 @@
+<div className={'tag tag-sm mb-24 tag-' + (props.meta.tags[0] == 'package:HTML' ? "" : "yellow")}>
```

Comment:
Also, because the "package:HTML" is not always on position 0 you can use: 
```suggestion
<div className={'tag tag-sm mb-24 tag-' + (props.meta.tags.includes('package:HTML') ? "" : "yellow")}>
```